### ISSUE TITLE
logoff not re-routing to SSO in Chrome

### ISSUE THREAD

**Issue by Dibyanshu** [damienbod/angular-auth-oidc-client#848]
**Describe the bug**
So for the chrome browser oidcSecurityService.logoff() is not re-routing to the identity server, although the logout session is cleared and I can not access the API anymore which is perfect. For other browsers(Edge and Mozilla) it is redirecting to SSO login page. I am not sure what is going wrong, please somebody help.  
I am experiencing this problem in version 11.1.4 and in chrome(85.0.4183.83 (Official Build) (64-bit))

**To Reproduce**
1. From any angular app in the chrome browser, try to call the "oidcSecurityService.logoff()"  

**Expected behavior**
This should log you out and redirect to SSO URL login screen

**Screenshots**
I tried to debug it in chrome browser and in the below js file the line number 2895 condition got satisfied always for chrome
and goes to else condition for other browsers(line number 2901) - refer the screenshot below
/node_modules/angular-auth-oidc-client/__ivy_ngcc__/fesm5/angular-auth-oidc-client.js

![OIDC-bug](https://user-images.githubusercontent.com/4135654/91849065-c33dae80-ec5b-11ea-8353-bde8d66ea2f9.png)

**Platform and Browser information:**
 - OS: Windows 10
 - Browser Chrome 
 - Version 85.0.4183.83 (Official Build) (64-bit)

**Comment by Expelz** [comment]
Hello @Dibyanshu!
I guess that you invoke `logoff` function from `OidcSecurityService.checkSessionChanged$` or `PublicEventsService.registerForEvents()` subscription. If I'm right please use `logoffLocal` instead of `logoff` and then you can do redirect to any url.

If it's not your case then please describe in more details where and in what situation you invoke `logoff` function or share some code.

**Comment by Stefano95054** [comment]
In my case, the topic is reproduced with the flag enabled:
**startCheckSession: true**.
angular-auth-oidc-client version: **11.2.0**

Live example to reproduce:
**sample-code-flow-auto-login** from samples of this repo. After login, instead of redirection to the authorization page, an automatic login back occurs. In fact, it is impossible to log out correctly.

**Comment by jangaraj** [comment]
I was able to fix it. My problem (tested in the Firefox, Chrome) was a logout link:
```
<li class="nav-item"><a class="nav-link" *ngIf="isAuthenticated" href (click)="logout()">Logout</a></li>
```
Just `return false` from `logout` function:
```
    logout() {
        this.oidcSecurityService.logoff();
        // return false otherwise logout link is followed and IDP logout url execution is cancelled immediately
        return false;
    }
```
It will be nice to have it in the doc as well https://github.com/damienbod/angular-auth-oidc-client/blob/main/docs/logout.md

**Comment by Stefano95054** [comment]
@jangaraj Unfortunately your workaround doesn't work. Logout still don't nothing.

**Comment by FabianGosebrink** [comment]
Should be fixed with #967
